tce/CHANGES

```diff
@@ -10,6 +10,9 @@ Notable changes and features
   Can also emulate 16-bit aligned loads, but emulation of 16-bit unaligned
   loads not supported, so all code will not compile of 16-bit loads are
   left out.
+- Plugin compilation speedup by 50-60% by creating a pluginwrapperfile that
```

Comment:
Make it more specific "what plugin". Perhaps just "Compiler backend compilation speedup...".